persistence/sql/migrations/20191100000010_reauthentication.up.fizz

```diff
@@ -0,0 +1 @@
+add_column("selfservice_login_requests", "is_reauthentication", "bool", {default: false})
```

Comment:
I would like to rename this to `forced` or, alternatively `prompt` although I think that forced is clearer.

Reply:
+1 forced